src/Serilog.Sinks.AzureDataExplorer.Tests/Serilog.Sinks.AzureDataExplorer.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Do we want to use centralized package management feature in Nuget ? Makes it easier to maintain revisions

Reply:
done